### DISCUSSION TITLE
As for the unbounded type, what default lenght is?

### DISCUSSION THREAD

**Discussion by wjhtinger** [eProsima/Fast-DDS#3919] [Q&A]
I know the default lenght of string type is 255, but how about other container type, such as vector, map etc.

**Comment by JLBuenoLopez** [comment]
The other container types default length when not specified in the IDL is 100 preallocated elements (when using the default [MemoryManagementPolicy](https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/core/policy/eprosimaExtensions.html#memorymanagementpolicy))
